packages/launchpad/src/migration/MajorVersionWelcome.vue

```diff
@@ -9,64 +9,117 @@
         class="bg-white rounded-b max-h-72vh pb-90px overflow-scroll"
       >
         <div class="h-full">
-          <div class="p-16px">
+          <div
+            class="p-16px"
+            data-cy="release-highlights"
+          >
             <h1 class="font-medium mt-4px text-center mb-32px tracking-tighter text-22px text-gray-1000">
               {{ t('majorVersionWelcome.title') }}
             </h1>
             <div class="mb-16px">
               <ExternalLink
-                href="https://on.cypress.io/changelog#11-0-0"
+                href="https://on.cypress.io/changelog#12-0-0"
                 class="font-bold text-indigo-500"
               >
-                11.0.0
+                12.0.0
               </ExternalLink>
               <span class="font-light pl-10px text-gray-500 text-14px">
-                Released {{ versionReleaseDates['11'] }}
+                Released {{ versionReleaseDates['12'] }}
```

Comment:
We need to manually add the release date, I should put this in the docs. `versionReleaseDates` is a computed value in this same file.

Reply:
OH I was assuming this was coming from NPM. Good catch.

Reply:
👍  Mainly doing it this way so it works offline.

Reply:
[`c4975a8` (#24925)](https://github.com/cypress-io/cypress/pull/24925/commits/c4975a881898d30e15fb9d934e5de62ed6ded47c) 

//cc @mjhenkes FYI when we go to release - we will need to verify this date is correct.